tests/test_formatters.py

```diff
@@ -988,3 +991,31 @@ def test_add_language_divs_fr_does_not_replace(lang: str):
 )
 def test_remove_language_divs(input: str, output: str):
     assert remove_language_divs(input) == output
+
+
+@pytest.mark.parametrize(
+    "input,output_contains",
+    (
+        ("abc 123", ["abc 123"]),
+        ("Hi,\n[[ircc-ga-seal]]\nBye", ["Hi,", "<img", "Bye"]),
+        ("Hi,\n[[ircc-ga-seal]]\nBye[[ircc-ga-seal]]", ["Hi,", "<img", "Bye"]),
```

Comment:
Can we add the last <img so that we catch if the token doesn't get changed?

Reply:
Let me know if this addresses your comment: https://github.com/cds-snc/notification-utils/pull/99/commits/72965be902cb70d0d36f72f230f0eb0855daf8ce

Reply:
I believe this is now resolved based on the latest revisions.